packages/microsoft-reactnative-sampleapps/windows/SampleAppCS/MainPage.xaml

```diff
@@ -1,3 +1,5 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License.  -->
```

Comment:
>   [](start = 36, length = 2)

All Xml comments end with double-space... 
Or is that the auto-formatter?

Reply:
It was copy & paste. I can fix it.

---
In reply to: [432967124](https://github.com/microsoft/react-native-windows/pull/5071#discussion_r432967124) [](ancestors = 432967124)

Reply:
Removed extra space.

---
In reply to: [433240152](https://github.com/microsoft/react-native-windows/pull/5071#discussion_r433240152) [](ancestors = 433240152,432967124)